packages/ocamlformat-0-8/ocamlformat-0-8.0.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Josh Berdine <jjb@fb.com>"
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ocamlformat.git"
+url {
+  src: "https://github.com/ocaml-ppx/ocamlformat/archive/0.8.tar.gz"
+  checksum: [
+    "md5=c3462e7bc4176b4d1126403123a99dac"
+    "sha512=f0010926ccc5a8faa661d74b7b51bcd5fc65a23cfea4c9f3cf24d2998a46149e5f73775536d0432dd502d32bc9015ac12888edf6933ec2023c64cfb597a2bb36"
+  ]
+}
+license: "MIT"
+build: [
+  ["tools/gen_version.sh" "src/Version.ml" version] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
```

Comment:
This will try to build the `ocamlformat-0-8` package, which fails.  I think you need an extra two steps here: run the dune build with `-p ocamlformat`, and then rm the install file, and then manually copy the binary over to `ocamlformat-0.8` so that it doesnt clash with `ocamlformat`